webapp/channels/src/components/admin_console/admin_definition.jsx

```diff
@@ -6103,35 +6103,6 @@ const AdminDefinition = {
                         help_text_default: 'Allow users to select GIFs from the emoji picker via a Gfycat integration.',
```

Comment:
Can we remove the Gfycat reference here? 

`Allow users to select GIFs from the emoji picker`